translate/package.json

```diff
@@ -5,16 +5,12 @@
   "license": "Apache Version 2.0",
   "author": "Google Inc.",
   "scripts": {
-    "test": "mocha -R spec --require intelli-espower-loader ../test/_setup.js test/*.test.js",
-    "system-test": "mocha -R spec --require intelli-espower-loader ../system-test/_setup.js system-test/*.test.js"
+    "test": "cd ..; npm run st -- translate/system-test/*.test.js"
```

Comment:
Nit: I'm not a fan of `cd`ing in tests, especially if this affects the user's current directory.